### ISSUE TITLE
Row data not loading with getRows() function

### ISSUE THREAD

**Issue by prasanjit101** [theoephraim/node-google-spreadsheet#665]
![image](https://github.com/theoephraim/node-google-spreadsheet/assets/57594105/a133aaa0-f8c2-4636-8cdc-4b07a0f7a7f4)

I have a spreadsheet with data populated as above.
I am trying to load the row data. But complete data is not loaded. Only the first columns data are loaded for a row (as shown in below image).

![image](https://github.com/theoephraim/node-google-spreadsheet/assets/57594105/87eed08e-d130-4101-9b74-e405c13fb275)

**Comment by prasanjit101** [comment]
Yeah, in the SS the headerRow isn't set to 1 but with it even set to 4. The data is not loading.
Unmerging the cell might work but I wanted to retain the structure.
So the workaround that worked for me is using `getCells` instead of `getRows`.

**Comment by theoephraim** [comment]
without setting the header row to the row that contains the headers, it assumes you have a table of rows that only have 1 column - with the title "Marketing Activity Log".  I suspect if you get that set correctly, it should work!
